releasenotes/notes/Settings-endpoint-928ae5b6e8ddc625.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    Added a /test/settings endpoint to change the settings on the fly.
```

Comment:
little bit iffy about exposing the raw config values direct to users. This'll probably result in accidental breakages to downstream test suites that depend on config values that we might accidentally change in the test agent.

If we do wanna go this route then I'll want to ensure we have tests covering the config names we want to be stable.

Reply:
Done in https://github.com/DataDog/dd-apm-test-agent/pull/198/commits/faf6994bd04d4ec5ce8551545cb649454ef63d0e